.circleci/config.yml

```diff
@@ -345,6 +344,18 @@ jobs:
       - *slack_notify_on_failure
       - *save_cache
 
+  # -------------------------------------------------------------------------------------------
+  # Job to check that the commit message is formatted correctly.
+  # -------------------------------------------------------------------------------------------
+  check_commit_message:
```

Comment:
Rather than a different job which requires set up and tear down.

Could we place the logic in the `-run` block?

```yaml
- run:
    name: Check Commit Message (PR Only)
    command: |
      if [ -n "$CIRCLE_PR_NUMBER" ]; then
        yarn ng-dev commit-message validate-range --range << pipeline.git.base_revision >>...<<pipeline.git.revision>>
      fi

Reply:
Nice, I wanted to do something similar, but I thought that we wanted to keep these scripts as one-liners. Changed.

Reply:
Definitely want to keep minimal complexity, but I think this is less complex (and expensive) than having a whole separate job.